src/linter.ts

```diff
@@ -76,5 +78,32 @@ export const lintNode = (
     );
   }
 
-  return results;
+  const isAKnownException = (violation: IRuleResult, locations: IExceptionLocation[]): boolean => {
```

Comment:
Could we move this function out of `lintNode`? Doesn't seem like it accesses anything out of its own scope.

Reply:
@P0lip Fixed